README.md

```diff
@@ -18,64 +19,64 @@ Status](https://travis-ci.org/puppetlabs/puppetlabs-aws.svg?branch=master)](http
 ##Description
 
 Amazon Web Services exposes a powerful API for creating and managing
-its Infrastructure as a Service platform. This module
-allows you to drive that API using Puppet code. In the simplest case
-this allows you to create new EC2 instances from Puppet code, but more
-importantly it allows you to describe your entire AWS infrastructure and
+its Infrastructure as a Service platform. The aws module
+allows you to drive that API using Puppet code. In the simplest case,
+this allows you to create new EC2 instances from Puppet code. More
+importantly, it allows you to describe your entire AWS infrastructure and
 to model the relationships between different components.
 
 ##Setup
 
-The module relies on the Amazon AWS Ruby SDK, so first install this. The
-SDK is available as a gem so install it into the same Ruby as used by
-Puppet.
+###Requirements
 
-    gem install aws-sdk-core
+* Puppet 3.4 or greater
+* Ruby 1.9
+* Amazon AWS Ruby SDK (available as a gem)
+* Retries gem
 
-We also use the retries library so install that with:
+###Installing the aws module
```

Comment:
Indeed we should! Good catch, thank you. H1 and H2 are leading caps, the rest should be sentence case.